### ISSUE TITLE
InsiderMobileAdvancedNotification/InsiderPushNotification.h not found in PROJECTNAME-Bridging-Header.h

### ISSUE THREAD

**Issue by hwq0521** [useinsider/reactjs-sample#3]
Hi, I'm facing error of InsiderMobileAdvancedNotification/InsiderPushNotification.h not found. May I know what's the problem? Even I tried add path in Header Search Paths and User Header Search Paths, still not working.

<img width="1315" alt="Screenshot 2021-01-13 at 5 55 17 PM" src="https://user-images.githubusercontent.com/74752520/104436637-d036bb00-55c8-11eb-892c-d51cce6829a8.png">

**Comment by canberksinangil** [comment]
I have transferred the issue to correct repo, please refer our wizard which will order all the necessary steps so you can follow it. I also suggest you to clone this project and check this out 
https://github.com/useinsider/reactjs-sample/issues/1#issuecomment-740538912

**Comment by hwq0521** [comment]
Hi @canberksinangil , can you send me the correct repo? so I can follow up this issue. Thanks

**Comment by hwq0521** [comment]
Hi I have cloned the demo project. After npm install and pod install, then I run the project in Xcode, has error occured.
<img width="1497" alt="Screenshot 2021-01-15 at 9 44 57 AM" src="https://user-images.githubusercontent.com/74752520/104670994-da1b0400-5717-11eb-886c-feb4091a6237.png">

I also found that your project don't have **bridging-header.h** file, is it no need create? Because I follow the link setup, is it required step, and there is the place the **InsiderPushNotification.h** not found.
